client/src/views/WorkspaceDetail.vue

```diff
@@ -69,7 +69,11 @@ export default {
           graphs { name }
         }
       }`});
-      this.tables = response.data.data.workspaces[0].tables.map(table => table.name);
+      this.tables = response.data.data.workspaces[0].tables.map(table => {
+        let name = table.name;
+        let displayName = name.includes("_") ? name.replace(/_/g, " ") : name;
+        return {name: table.name, display:displayName }
```

Comment:
Would be better to format this more clearly like

```javascript
return {
  name: table.name,
  display: displayName
};
```

Reply:
My plan was to leave the graph side alone until we had the functionality to upload/view the graphs but I can definitely add the styling changes to the available graphs consistent with the tables

Reply:
added styling to reflect table list

Reply:
This reverted to the original mapping so this return statement doesn't exist.